internal/gps/source_cache_bolt.go

```diff
@@ -73,31 +74,31 @@ func (c *boltCache) close() error {
 //
 // 1) Versions buckets hold version keys with revision values:
 //
-//	Bucket: "versions:<timestamp>"
-//	Keys: "branch:<branch>", "defaultBranch:<branch>", "ver:<version>"
+//	Bucket: "v<timestamp>"
```

Comment:
The primary motivation was being able to delete entire buckets in a single op, rather than prefix scanning and deleting each key.  Another was avoiding repeated, long, complex key-prefixes.  Perhaps the `Revision-versions` buckets have few enough entries to yield speed up from prefixes, but most other buckets are quite large and/or would have keys much longer than their values.